Supernotes/script.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "Supernotes",
 	"script": "Supernotes.js",
-	"version": "0.0.4",
+	"version": "0.0.5",
```

Comment:
The norm is using the previousversions key for cases like this

`   "version": "1.02",
    "previousversions": ["1.0", "1.01"],`